helix-core/src/main/java/org/apache/helix/manager/zk/ZkCallbackCache.java

```diff
@@ -220,7 +220,7 @@ public void handleStateChanged(KeeperState state) throws Exception {
   }
 
   @Override
-  public void handleNewSession() throws Exception {
+  public void handleNewSession(final String sessionId) throws Exception {
```

Comment:
I know we already reached an agreement not to keep the old method signature, but let's double check - users shouldn't be extending this class, right? Should we still try to make this backward-compatible in case users are using this public method?

Reply:
My understanding is, if users decide to import the new listener, users should realize that handleNewSession() is not available and they should use handleNewSession(sessionId). If users are using the old public method, they are forced to replace it with the new method.